internal/unionstore/memdb.go

```diff
@@ -163,6 +164,8 @@ func (db *MemDB) Reset() {
 	db.vlogInvalid = false
 	db.size = 0
 	db.count = 0
+	metrics.MemdbSize.Set(0.0)
+	metrics.MemdbCount.Set(0.0)
```

Comment:
Sorry I didn't understand it well. The metrics looks like global gauges. May there be many MemDB instances in a single process and they all `Set` values to the single gauge by themselves?

Reply:
I've mentioned it above and discussed it with @longfangsong , it's needed to be changed I think.